Gaijinizer patch/Map062.txt

```diff
@@ -191,115 +203,127 @@ human maternal love.#MTLed
 どんなに甘えてくれる…私を受け入れてくれる『ボウヤ』も
  ほんの僅かにしか、一緒にいられない……。
 > CONTEXT: Map062/events/5/pages/1/50/Dialogue/enemy4
-I'm sure I'm not satisfied. No matter how sweet
-you are..."boy" who accepts me
-Only a few, I can not be together ....#MTLed
+I'm certain I cannot be satisfied...
+
+No matter how many let me spoil them, there are
+very few "little boys" who will accept me and stay
+together with me...#RonIffy
```

Comment:
Minor flow:
>_**No matter how many let me spoil them, there are
very few "little boys" who will accept and stay
together with me...**_